content/blogs/gui-setup-part1-setupapi.md

```diff
@@ -0,0 +1,47 @@
+---
+title:      "1st-stage GUI setup - September 2023: partly Wine-syncing setupapi"
+author:     "hbelusca"
+date:       2023-10-30
+tags:       [ newsletter, gui-setup ]
+---
+
+Greetings to all ReactOS followers!
+As many of you certainly know by now, I have been officially hired by the ReactOS Deutschland e.V. to develop the graphical version of the 1st-stage ReactOS installer ("1st-stage GUI setup").
```

Comment:
I think it would be better to elaborate a bit more on this, or link to another blog post explaining it in detail.

Reply:
Will do

Reply:
works for me.